tests/test_config_schema.py

```diff
@@ -39,12 +39,15 @@ async def test_config_schema(self) -> None:
             dev_type=common.DeviceType.FTDI.value,
             dev_id="ABC",
             sens_type=common.SensorType.TEMPERATURE.value,
+            location="bla:bla:bla:bla",
```

Comment:
If you switch to comma separated values (as per my review of ts_config_ocs) please update everywhere in this package.

Reply:
Done